components/lib/VButton.vue

```diff
@@ -126,7 +126,7 @@ a:not([disabled='true']):hover {
   color: #00519c;
 }
 a:not([disabled='true']):active {
-  transform: scale(90%);
+  transform: scale(99%);
```

Comment:
Prefer a unitless value for scales.

```suggestion
  transform: scale(0.99);
```

Reply:
@punaifang added changes